loops.js

```diff
@@ -87,5 +107,16 @@
 // etc.
 // ==========================================
 
-
+console.log("Opdracht 4");
+for (let i = 1; i <= 100; i++) {
+    if (i % 3 === 0 && i % 5 !== 0) {
+        console.log("Fizz")
+    } else if (i % 5 === 0 && i % 3 !== 0) {
+        console.log("Buzz")
+    } else if (i % 3 === 0 && i % 5 === 0) {
+        console.log("FizzBuzz")
+    } else {
+        console.log(i)
+    }
+}
 
```

Comment:
Dit zou je nog simpeler kunnen maken;

for (let i = 1; i <= 100; i++) {
    if (i % 3 === 0 && i % 5 === 0) {
        console.log("FizzBuzz");
    }
    else if (i % 3 === 0) {
        console.log("Fizz");
    }
    else if (i % 5 === 0) {
        console.log("Buzz");
    }
    else {
        console.log(i);
    }
}